src/js/main.es6

```diff
@@ -0,0 +1,77 @@
+$(document).ready(function () {
+  let modal = function Modal() {
```

Comment:
why not just `function Modal` ? And why uppercase, is it a constructor function?